integration-tests/pkg/collector/introspection.go

```diff
@@ -23,3 +30,50 @@ func (k *K8sCollectorManager) IntrospectionQuery(endpoint string) ([]byte, error
 	body, err := io.ReadAll(resp.Body)
 	return body, err
 }
+
+func QueryConfig(t *testing.T, collectorIP string) []byte {
+	log.Info("Querying: /state/config")
+	body, err := IntrospectionQuery(collectorIP, "/state/config")
+	assert.NoError(t, err)
+	log.Info("Response: %q", body)
+	return body
+}
```

Comment:
I'm not 100% sold this is something we need, from a test we could just do:
```go
body, err := IntrospectionQuery(s.Collector().IP(), "/state/config")
s.Require().NoError(err)
```
This is simple enough and you can add the logging if you deem it necessary.

Reply:
Done